gvmd/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN set -ex; \
     cd /; \
     apt download openvas; \
     dpkg --fsys-tarfile openvas_*.deb | tar xf - ./usr/bin/greenbone-nvt-sync; \
-    sed -i 's/if \[ "`id -u`" -eq "0" \]/if fasle \&\& [ "`id -u`" -eq "0" ]/' /usr/bin/greenbone-nvt-sync; \
+    sed -i 's/if \[ "`id -u`" -eq "0" \]/if fasle \&\& [ "`id -u`" -eq "0" ]/' /usr/bin/greenbone-nvt-sync --rsync; \
```

Comment:
This will pass `--rsync` to `sed` command and throw an error message. The `/usr/bin/greenbone-nvt-sync` in this line is just the path of the file that we are modifying with `sed`